entries/p/paypal.com.json

```
@@ -4,7 +4,8 @@
       "paypal-search.com"
     ],
     "mfa": "allowed",
```

Passkeys seems to only be used by PayPal to replace the password.
```suggestion
    "passwordless": "allowed",
```